sql/xsql/src/main/scala/org/apache/spark/sql/xsql/XSQLExternalCatalog.scala

```diff
@@ -675,7 +675,7 @@ private[xsql] class XSQLExternalCatalog(conf: SparkConf, hadoopConf: Configurati
         case ELASTICSEARCH => CatalogTableType.TYPE
         case MONGO => CatalogTableType.COLLECTION
         case HBASE => CatalogTableType.HBASE
-        case MYSQL | ORACLE => CatalogTableType.JDBC
+        case MYSQL | JDBC => CatalogTableType.JDBC
         case DRUID => CatalogTableType.JDBC
```

Comment:
`case JDBC => CatalogTableType.JDBC` ?

Reply:
I haven't moved mysql to the jdbc module in this PR. So l didn't remove MYSQL for MysqlManager could work normally.